.pycodestyle

```diff
@@ -0,0 +1,6 @@
+[pycodestyle]
+count = False
+ignore = E226,E401,E402,W504
```

Comment:
Can we use the human-readable tags on these so we know what we're ignoring? With pylint, that's a possibility. Here are the codes:

E226 (*)	missing whitespace around arithmetic operator
E401	multiple imports on one line
E402	module level import not at top of file
W504 (*)	line break after binary operator

I feel that all of these are important for consistency and maintenance. I have some reasons:

E401 -- It's easier to see line additions/subtractions than line modifications when reviewing code.
E402 -- If imports are scattered through the file, it's hard to pin down our exact requirements when maintaining our environment files.
W504 and E226 -- They help visually parse different parts of a more involved equation.

Reply:
I have added both pylint and pycodestyle.
The codes in this file are just that, and pycodestyle maintains a list and their meanings. 
 This file is not meant to be updated once the developers are settled on what norms are expected in the project.
When `pycodestyle` is run, it will print the code and the error.

I have also found `pylint` while nice, is probably an overkill for a project of this size. But, I am ok with using `pylint` if that is the consensus

Reply:
If we want to run both of them, then we'd need to maintain the same "ignore" list and .rc file for both of them. Pylint will likely error out on these pycodestyle ignored rules if we don't maintain both .rc files. It might be a good reason to just pick one.

I'm not sure why pylint is overkill on its own?

Reply:
removed pycodestyle.  its all pylint now.